base_user_role/models/role.py

```diff
@@ -27,12 +38,6 @@ class ResUsersRole(models.Model):
     user_ids = fields.One2many(
         comodel_name="res.users", string="Users list", compute="_compute_user_ids"
     )
-    group_category_id = fields.Many2one(
-        related="group_id.category_id",
-        default=lambda cls: cls.env.ref("base_user_role.ir_module_category_role").id,
-        string="Associated category",
-        help="Associated group's category",
-    )
```

Comment:
This code was working well before, so I suppose the issue is that the default value is not set anymore because starting from Odoo 12.0 the related fields are `readonly` by default. Try with `readonly=False` here and check if it solves the issue?

Reply:
Setting the 'readonly' attribute to 'False" indeed works.